tools/run_tests/performance/bq_upload_result.py

```diff
@@ -157,6 +169,40 @@ def _populate_metadata_inplace(scenario_result):
     scenario_result['metadata'] = metadata
 
 
+def _populate_metadata_from_file(scenario_result, test_metadata_file):
+    with open(test_metadata_file, 'r') as f:
+        test_metadata = json.loads(f.read())
+
+    """eliminate managedField from metadata set"""
+    if 'managedFields' in test_metadata:
+        del test_metadata['managedFields']
+    annotations = test_metadata['annotations']
```

Comment:
I would move this line right before the if statement that delets the annotation (i.e., after the comment). It's one logical block in the code.